modules/private-clusters-about-gcp.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * installing/
+
+[id="private-clusters-about-gcp_{context}"]
+= Private clusters in GCP
+
+To create a private cluster on Google Cloud Platform (GCP), you must provide an existing private VPC and subnets to host the cluster. The installation program must also be able to resolve the DNS records that the cluster requires. The installation program configures the Ingress Operator and API server for only internal traffic.
+
+The cluster still requires access to Internet to access the GCP APIs.
+
+The following items are not required or created when you install a private cluster:
+
+* Public subnets
+* Public network load balancers, which support public ingress
+* A public DNS zone that matches the `baseDomain` for the cluster
+
+The installation program does use the `baseDomain` that you specify to create a private DNS zone and the required records for the cluster. The cluster is configured so that the Operators do not create public records for the cluster and all cluster machines are placed in the private subnets that you specify.
+
+Because it is not possible to limit access to external load balancers based on source tags, the private cluster uses internal load balancers to allow access to internal instances instead of network load balancers.
```

Comment:
I believe we now use internal load balancers in GCP for both private and public clusters. The difference is that only public clusters create the external network load balancers.